gitlab/resource_gitlab_project_freeze_period_test.go

```diff
@@ -0,0 +1,176 @@
+package gitlab
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func TestAccGitlabProjectFreezePeriod_basic(t *testing.T) {
+	var schedule gitlab.FreezePeriod
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabProjectFreezePeriodDestroy,
```

Comment:
I think `testAccCheckGitlabProjectFreezePeriodDestroy` is checking to make sure the Project is destroyed.  Is that correct? You can use `testAccCheckGitlabProjectDestroy` so you don't have to make your own method.

Checking to see if the freeze period was deleted could be a step in the test.  Here's an example: https://github.com/gitlabhq/terraform-provider-gitlab/blob/master/gitlab/resource_gitlab_project_share_group_test.go#L31

I don't see the delete step used consistently in our tests so I'm not sure what's best practice.